cmd/cosign/cli/options/signblob.go

```diff
@@ -22,16 +22,18 @@ import (
 )
 
 // SignBlobOptions is the top level wrapper for the sign-blob command.
+// The new output-certificate flag is only in use when COSIGN_EXPERIMENTAL is enabled
```

Comment:
@asraa  is this docstring on the right position? I thought about adding it to the `SignBlob` function, but the SignBlob function has a `// nolint` on top.. I am not sure if I can have docstrings and nolint at the same time.